README.md

```diff
@@ -1,22 +1,22 @@
-# gosplat (Go Static Package Language-model Analysis Tool)
+# gosplat (GoLang Static Package Language-model Analysis Tool)
 ## Description
-A tool which can find lacking file structure. It checks if functions are in the right packages, can give possible packages to move functions into if they have a low match, and give hints to packages with bad naming.
+A tool to analyze packages and functions in GoLang and finding possible improvements to which package functions are placed in.
 
 ### Dynamic analysis which curates its choices to your code base
-Gosplat uses the FastText model trained on a large number of open-source projects in GoLang in a curated manner so that it's able to find ossible inconsistencies in your program structure. 
+Gosplat uses the FastText model trained on a large number of open-source projects in GoLang in a curated manner so that it's able to find possible inconsistencies in your program structure. It checks if functions are in the right package, and can give possible packages to move functions to if they have a low match, also giving hints to packages with bad naming.
 
 ### Doesn't steal any data! :D
-The FastText model is pre-trained and of course, will not by any means save any data of the projects it's used in. The model is downloaded from our gitlab repo and ready to be used afterwards.
+The FastText model is pre-trained and of course, will not by any means save any data of the projects it's used in. The model is downloaded from a GitLab repo and ready to be used afterwards.
 
 ## Goals of this project
-Gosplat's and our goals are to create a nice, quick and easy way to analyze possible holes in your or your team's file structure. It may give helpful hints which can aid in creating a more maintainable enviorment for future projects to come!
+Gosplat's and our goals are to create a nice, quick and easy way to analyze possible holes in your or your team's file structure. It may give helpful hints which can aid in creating a more maintainable environment for future projects to come!
```

Comment:
Not your fault, mine but it's "you or your team's file structure", changing "your" to "you"